src/test/integration/integrationtests.py

```diff
@@ -167,18 +201,89 @@ def test_self_link(self):
             self.assertEqual(person['links']['self'], api_url + osu_id)
 
         person_id_res = utils.get_person_by_osu_id(osu_id)
-        self.assertEqual(person_id_res.json()['data']['links']['self'], api_url + osu_id)
+        self.assertEqual(person_id_res.json()['data']['links']['self'],
+                         api_url + osu_id)
 
         jobs_res = utils.get_jobs_by_osu_id(jobs_osu_id)
         for job in jobs_res.json()['data']:
             attributes = job['attributes']
             position_number = attributes['positionNumber']
             suffix = attributes['suffix']
             expected_url = "{}{}/jobs?positionNumber={}&suffix={}".format(
-                    api_url, osu_id, position_number, suffix)
+                api_url, osu_id, position_number, suffix)
 
             self.assertEqual(job['links']['self'], expected_url)
 
+    def test_name_search(self):
+        person_res = utils.get_person_by_ids({
+            'firstName':
+            regular_name_person['first_name'],
+            'lastName':
+            regular_name_person['last_name']
+        })
```

Comment:
I think these lines aren't long enough to break to new lines. Could you change the code like you did in [line 261](https://github.com/osu-mist/persons-api/blob/89462c618541f3706117dd844adb532edb3d23e9/src/test/integration/integrationtests.py#L264-L267)?

```python
person_res = utils.get_person_by_ids({
    'firstName': regular_name_person['first_name'],
    'lastName': regular_name_person['last_name']
})
```